### PR TITLE
16 : Moved Client, Event, PhysicalLocation domain classes and their d…

### PR DESCRIPTION
**Pull request by rehammuzzamil**
…ependent classes to opensrp-plan-evaluator library.
https://github.com/OpenSRP/opensrp-plan-evaluator/issues/16

### CODE REVIEW

**Comment by githengi** [comment]
update plan evaluator version

**Comment by rehammuzzamil** [comment]
> update plan evaluator version

Done

File: src/test/java/org/opensrp/domain/AddressTest.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.Warning;
+import org.smartregister.domain.Address;
 
 public class AddressTest {
```
Comment (githengi) [inline src/test/java/org/opensrp/domain/AddressTest.java:22]: Test should be moved

File: src/test/java/org/opensrp/domain/ObsTest.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.Warning;
+import org.smartregister.domain.Obs;
 
 public class ObsTest {
```
Comment (githengi) [inline src/test/java/org/opensrp/domain/ObsTest.java:22]: Test should be moved

Comment (rehammuzzamil) [reply src/test/java/org/opensrp/domain/ObsTest.java]: Done

File: src/test/java/org/opensrp/domain/AddressTest.java (hunk 1 above)
Comment (rehammuzzamil) [reply src/test/java/org/opensrp/domain/AddressTest.java]: Done

**Pull request merged**